azkaban-common/src/test/java/azkaban/executor/InteractiveTestJob.java

```diff
@@ -141,4 +142,10 @@ public void cancel() throws InterruptedException {
     info("Killing job");
     failJob();
   }
+
+  public static void clearTestJobs(final String... names) {
```

Comment:
No need to sync, it's ensured by other constrains before this is ever called. And the map itself is a ConcurrentHashMap.

Running in parallel in the same JVM? Then the static fields should be converted into instance fields, but otherwise should be fine.